batch/sql/cleanup-add-batch-updates.sql

```diff
@@ -0,0 +1,21 @@
+START TRANSACTION;
```

Comment:
Didn't know where exactly to put this, but this migration is going to take on the order of an hour.